lib/gooddata/lcm/lcm2.rb

```diff
@@ -353,9 +353,15 @@ def perform(mode, params = {})
           success: errors.empty?
         }
 
-        has_errors = errors.any?
-        # Fail whole execution if there is any failed action
-        fail(JSON.pretty_generate(errors)) if strict_mode && has_errors
+        if errors.any?
+          error_message = JSON.pretty_generate(errors)
+          GoodData.logger.info(error_message)
```

Comment:
Maybe the log level should be error and not info....

Reply:
Good point, fixed.